packages/form-builder/src/blocks/fields/field.js

```diff
@@ -92,6 +110,11 @@ const field = {
                 source: 'attribute',
                 default: false,
             },
+            displayInAdmin: {
+                type: 'boolean',
+                source: 'attribute',
+                default: false,
```

Comment:
I'm thinking custom fields should be `true` by default to display in admin & receipt.  It's something I see being easily missed by the admin and then immediately confused by lol

Reply:
I went with `false` as the default to [maintain parity with the Fields API](https://github.com/impress-org/givewp/pull/5886) and am open to discussing/considering further.

Reply:
Having slept on it, I agree that `true` would be a better default for this use case.

Resolved, see https://github.com/impress-org/givewp-next-gen/pull/99/commits/2badd835bdfd5df0a772112873e631a63c2c6dc6.